admin_guide/limit_pod_access_egress.adoc

```diff
@@ -0,0 +1,117 @@
+the addresses that some or all pods can access from within the cluster, so that:
+
+- A pod can only talk to internal hosts, and cannot initiate connections to the
+public Internet.
++
+Or, 
+- A pod can only talk to the public Internet, and cannot initiate connections to
+internal hosts (outside the cluster).
++
+Or,
+- A pod cannot reach specified internal subnets/hosts that it should have no
+reason to contact.
+
+Project administrators can neither create `*EgressNetworkPolicy*` objects nor
+edit the ones you create in their project. 
+
+[CAUTION]
+====
+You must have the 
+xref:../install_config/configuring_sdn.adoc#install-config-configuring-sdn[*ovs-multitenant* plug-in] enabled in order to limit pod access via egress policy.
+====
+
+[[admin-guide-config-pod-access]]
+== Configuring Pod Access Limits
+
+To configure pod access limits, you must use the `oc` command or the REST API.
+You can use `oc [create|replace|delete]` to manipulate `*EgressNetworkPolicy*`
+objects. The *_api/swagger-spec/oapi-v1.json_* file has API-level details on how
+the objects actually work. 
+
+. Navigate to the project you want to affect. 
+. Create a JSON file for the pod limit policy:
++
+----
+# oc create -f <policy>.json
+----
+. Configure the JSON file with policy details. For example:
++
+----
+{
+    "kind": "EgressNetworkPolicy",
+    "apiVersion": "v1",
+    "metadata": {
+        "name": "default"
+    },
+    "spec": {
+        "egress": [
+            {
+                "type": "Allow",
+                "to": {
+                    "cidrSelector": "1.2.3.0/24"
+                }
+            },
+            {
+                "type": "Deny",
+                "to": {
+                    "cidrSelector": "0.0.0.0/32"
+                }
+            }
+        ]
+    }
+}
+----
++
+When the example above is added in a project, it allows traffic to `1.2.3.0/24`,
+but denies access to all other external IP addresses. This would not affect
+traffic to other pods.
+
+[NOTE]
+====
+The `default` project (and any other global namespace) cannot have egress
+policy. 
+
+If you merge two projects together (via `oadm pod-network join-projects`), then you cannot use egress policy in _any_ of the joined projects.
+====
+
+== Examples and Restrictions
+
+- You can create `*EgressNetworkPolicy*` objects with rules that
+allow or deny outgoing pod traffic to specific CIDR ranges. For example: 
+`10.0.0.0/8`.
+
+- Different projects can have different `*EgressNetworkPolicies*`, allowing you 
+to allow `<project A>` access a specified IP range, but deny the same access to 
+`<project B>`. 
+
+- If you merge two projects together via `oadm pod-network join-projects`, then
+you cannot have an `*EgressNetworkPolicy*` object in either namespace. If you
+do, it logs an error and then adds a rule dropping all external traffic from
+both projects. This happens regardless of the order in which you merge the
+projects and create the policy object. 
```

Comment:
The exact semantics of what happens when you try to join projects that use egress policy is undefined. Just say that you can't do it. (The Trello card specifies what happens in detail because we wanted QE to check that the expected thing happens.)